cmake/projects/F2C/hunter.cmake

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2015-2016, University College London
+# All rights reserved.
+include(hunter_add_version)
+include(hunter_download)
+include(hunter_pick_scheme)
+include(hunter_add_package)
+include(hunter_configuration_types)
+
+# Makes it possible to use syste cfitsio
+hunter_add_version(
+    PACKAGE_NAME
+    F2C
+    VERSION
+    "0.0"
+    URL
+    "http://netlib.sandia.gov/f2c/libf2c.zip"
```

Comment:
It should be. It should last as long as the US national labs do.
But it won't be have version info, because f2c is not released with versions, AFAIK.